packages/firebase_storage/lib/src/firebase_storage.dart

```diff
@@ -125,6 +125,20 @@ class FirebaseStorage {
       'time': time,
     });
   }
+
+  Future<StorageReference> getReferenceFromUrl(String fullUrl) async {
+    String path = await channel.invokeMethod(
+        "FirebaseStorage#getReferenceFromUrl", <String, dynamic>{
+      'app': app?.name,
```

Comment:
`app` and `bucket` seems not used in the platform code.

Reply:
They are used in the `getStorage:` method (on iOS) and at the beginning of onMethodCall (on Android) to inflate the `FirebaseStorage` object on the native side.